### PR TITLE
Html layout (Don't merge)

### PR DESCRIPTION
**Pull request by Moejoe90**
Currently the file modified is Index.html with all react components removed to represent a pure html environment.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,14 @@ sudo: false
 language: node_js
 node_js:
   - "7"
+notifications:
+  email: false
```
Comment (recmo) [inline .travis.yml:7]: What is this change for?

File: index CPY.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<!doctype html>
```
Comment (recmo) [inline index CPY.html:1]: Spaces in file names cause all sorts of problems. It's better to use `_`.

File: index CPY.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>To do</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.css" type="text/css">
```
Comment (recmo) [inline index CPY.html:20]: If this was not a test file, it should either have and `integrity` attribute or be included in the source (we already have `flexbox` as a dependency)

[No action required]

File: public/index.html [hunk 4]
```diff
@@ -15,14 +15,119 @@
     -->
     <title>To do</title>
     <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.css" type="text/css">
```
Comment (recmo) [inline public/index.html:20]: We should use a bundled version, the one that was added in `package.json`.

@banciur How does one include stylesheet like that? `import 'flexboxgrind';` in `index.js`?

**Review by recmo — CHANGES_REQUESTED** [review]
Somehow `src/index.js` got reverted to an older version. I think there's something wrong in the commit history.

File: .travis.yml (hunk 1 above)
Comment (Moejoe90) [reply .travis.yml:7]: This is to disable Travis email notification, apparently you have to manually put this in the travis.yml file

File: index CPY.html (hunk 2 above)
Comment (Moejoe90) [reply index CPY.html:1]: Will take that into account

**Comment by recmo** [comment]
This PR has been superseded by others.

**Pull request closed without merging**